devices/xc7a100t/CMakeLists.txt

```diff
@@ -12,3 +12,10 @@ generate_chipdb(
     device_config ${PYTHON_INTERCHANGE_PATH}/test_data/series7_device_config.yaml
     test_package csg324
 )
+
+generate_xc7_device_with_timings(
+    device xc7a100t
+    input_device ${xc7a100t_target}
+    output_device xc7a100t_timing_target
+    family artix7
+)
```

Comment:
Rather than having to add the timings generation function here, it might be better to add this to the `generate_xc7_device_db`, so that the `timing_device` becomes the final output device used to generate also the chipdb

Reply:
Moved to generate_xc7_device_db